### DISCUSSION TITLE
Monoliths and Microservices | System Design

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#56] [Comments]
# Monoliths and Microservices | System Design

In this tutorial, we will discuss Monoliths and Microservices

https://www.karanpratapsingh.com/courses/system-design/monoliths-microservices

**Comment by divyanshbhatt527** [comment]
Brilliantly explained the difference between the two, easy to understand with proper references wherever needed. 
Thanks Karan for this article.
